qutebrowser/config/configexc.py

```diff
@@ -92,6 +92,10 @@ class ConfigErrorDesc:
     traceback = attr.ib(None)
 
     def __str__(self):
+        if self.traceback:
+            return '{} - {}: {} '.format(self.text,
```

Comment:
Why the trailing space (also in `test_configexc.py`)?

Reply:
Oh oops!